HPE3PARPSToolkit-CLI.psm1

```diff
@@ -34380,11 +34379,2491 @@ Function Stop-3parWsapi()
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+##v2.3 
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+
+##########################################################################
+######################### FUNCTION Set-3parDomain #########################
+##########################################################################
+Function Set-3parDomain()
+{
+<#
+  .SYNOPSIS
+   Set-3parDomain Change currentdomain CLI environment parameter.
+
+  .DESCRIPTION
+   The Set-3parDomain command changes the currentdomain CLI environment parameter.
+
+  .EXAMPLE
+   Set-3parDomain
+   
+  .EXAMPLE
+   Set-3parDomain -Domain "XXX"
+   
+  .PARAMETER Domain
+	Name of the domain to be set as the working domain for the current CLI session.  
+	If the <domain> parameter is not present or is equal to -unset then the working domain is set to no current domain.
+	
+
+  .Notes
+    NAME: Set-3parDomain
+    LASTEDIT 14-03-2019 14:24:44
+    KEYWORDS: 3parVersion
+  
+  .Link
+    Http://www.hpe.com
+
+ #Requires PS -Version 3.0
+#>
+ [CmdletBinding()]
+ param(
+ [Parameter(Position=0, Mandatory=$false)]
+ [System.String]
+ $Domain,
+
+ [Parameter(Position=1, Mandatory=$false, ValueFromPipeline=$true)]
+ $SANConnection = $global:SANConnection
+ )
+
+ Write-DebugLog "Start: In Set-3parDomain - validating input values" $Debug 
+ #check if connection object contents are null/empty
+ if(!$SANConnection)
+ {
+  #check if connection object contents are null/empty
+  $Validate1 = Test-ConnectionObject $SANConnection
+  if($Validate1 -eq "Failed")
+  {
+    #check if global connection object contents are null/empty
+    $Validate2 = Test-ConnectionObject $global:SANConnection
+    if($Validate2 -eq "Failed")
+    {
+        Write-DebugLog "Connection object is null/empty or Connection object UserName,password,IPAaddress are null/empty. Create a valid connection object using New-SANConnection" " ERR: "
+        Write-DebugLog "Stop: Exiting Set-3parDomain since SAN connection object values are null/empty" $Debug 
+        Return "FAILURE : Exiting Set-3parDomain since SAN connection object values are null/empty"
+    }
+  }
+ }
+
+ $plinkresult = Test-PARCli -SANConnection $SANConnection
+ if($plinkresult -match "FAILURE :")
+ {
+   write-debuglog "$plinkresult"
+   Return $plinkresult
+ }
+
+ $Cmd = " changedomain "
+
+ if($Domain)
+ {
+	$Cmd += " $Domain "
+ }
+
+ $Result = Invoke-3parCLICmd -Connection $SANConnection -cmds  $Cmd
+ Write-DebugLog "Executing Function : Set-3parDomain Command" INFO: 
+ IF ([System.String]::IsNullOrEmpty($Domain))
+ {
+	$Result = "Working domain is set to no current domain."
```

Comment:
May may need to re-frame the sentence "Working domain is set to no current domain."

Reply:
Cmdlet is use to Set or Unset current domain.

if domain name is there then it will set it as Current Domain.
if domain name is Null the it will unset the current-domain.

change the statement to "Working domain is unset to current domain"